jobs/mobile-developer.md

```diff
@@ -0,0 +1,60 @@
+# Mobile Developer (Android/iOS)
+
+We are looking for experienced and initiative Mobile Developers to grow our team and develop meaningful products together. We value openness and 
+knowledge sharing, believe that communication is a key and develop the remote culture for around 6 years. We would like to work with someone who shares our values,
+acts independently and proactively.
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or ready to improve their skills) in React Native. We work with Kotlin + RxJava + Dagger for Android
+and Swift + RxSwift for iOS. We also use React Native and going to use it more
```

Comment:
The tech stack is inaccurate. Let's better say this way:
>We primarily use Kotlin for Android and Swift for iOS, yet certain Java or Objective-C knowledge might be useful.

The reason for that is that `RxJava`, `RxSwift`, and `Dagger` are quite random technologies and they don't really cover represent the stack we are (or may be) using for our native projects. We should either broaden the list (which is not what we need here I suppose) or stop with just used languages. I've added a part about `Java` and `Objective-C` experience due to the fact that, for instance, the current native project has a good part of the code written in those, the previous small one we used to work with had been written in `Java` and `Objective-C` completely. Also, all the native insides of the `react-native` project are written in `Java` and `Objective-C` still.
TL;DR: the ability to at least read and somehow maintain code in `Java` and `Objective-C` might be if not required but a useful skill to have.